boards/risc-v/bl602/bl602evb/src/bl602_bringup.c

```diff
@@ -62,8 +71,7 @@ int bl602_bringup(void)
   ret = mount(NULL, "/proc", "procfs", 0, NULL);
   if (ret < 0)
     {
-      syslog(
-        LOG_ERR, "ERROR: Failed to mount procfs at %s: %d\n", "/proc", ret);
+      ferr("ERROR: Failed to mount procfs at %s: %d\n", "/proc", ret);
```

Comment:
Inside the board bringup we should just be using the syslog as you did before.  Sorry I was not more clear.